CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ include(package_settings)
 include(add_enclave)
 
 # User configurable options
-option(USE_LIBSGX "Build oehost using SGX library requiring FLC" OFF)
+option(USE_LIBSGX "Build oehost using SGX library requiring FLC" ON)
```

Comment:
This breaks Windows by default right now. We should override by default in WIN32 instead of putting it on the user.

Reply:
See? This is why I had you review it, thanks 😉